tests/Utils/Models/Activity.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Utils\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\MorphTo;
+
+/**
+ * @property int $id
+ * @property int $activity_id
+ * @property string $activity_type
+ * @property \Carbon\Carbon $created_at
+ * @property \Carbon\Carbon $updated_at
```

Comment:
```suggestion
 * @property \Carbon\Carbon $updated_at
 *
 * Relations
 * @property-read \Tests\Utils\Models\Post|\Tests\Utils\Models\Task $content
 * @property-read \Tests\Utils\Models\User $user
```

Also, please add `|null` where appropriate.